src/main/java/org/cs1331/gitsubmitter/Utils.java

```diff
@@ -227,6 +229,32 @@ private static void decrypt(File file, File out, File aesIn,
         cos.close();
     }
 
+    /**
+     * Attempts to retrieve a unique identifier for this computer.
+     * The MAC address of the currently active network interface is used
+     * as this identifier. If the MAC address is unable to be retrieved,
+     * "000000000000" is returned.
+     * 
+     * @return A unique identifier of this computer
+     */
+    public static String getIdentifier() {
+        byte[] id;
```

Comment:
Better practice to set it to null explicitly as that's *what it should be right now.*

Readability: why make someone trace through your code to ensure that it's set to `null`? Sure, it might be a simple `try/catch` now, but it could get more complicated later.